CHANGELOG.md

```diff
@@ -9,3 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Migration from jCenter to Maven Central 🎉
 - ‼️ Important ‼️ groupId has changed. New groupId is `io.github.ackeecz`.
 - Package name changed as well from `cz.ackee.ankoconstraintlayout` to `io.github.ackeecz.ankoconstraintlayout`
+
+## **2.0.1**
```

Comment:
I think that there is some error in our communication, but how we can have unreleased stuff and 2.0.1 at the same time? :D It should be either part of the unreleased or the unreleased part should be released. Also please, keep the format as i said - [tag] - yyyy-MM-dd as a header

Reply:
I will rebase this and fix a changelog after previous version is released as stated here https://github.com/AckeeCZ/anko-constraint-layout/pull/7#discussion_r611054238